public/utils/constants.ts

```diff
@@ -380,3 +380,9 @@ export enum IndexOpBlocksType {
   MetaData = "9",
   ReadOnlyAllowDelete = "12",
 }
+
+export enum INDEX_OP_TARGET_TYPE {
+  INDEX = "index",
+  ALIAS = "alias",
+  DATA_STREAM = "datastream",
```

Comment:
Since we're using the enum value as modal title, we can change "alias" to "aliases" and change "datastream" to "data streams" to keep consistent with current wordings.

Reply:
Updated